### ISSUE TITLE
Add long break

### ISSUE THREAD

**Issue by dkandalov** [dkandalov/pomodoro-tm#5]

**Comment by dominikbulaj** [comment]
Any news here?

**Comment by dkandalov** [comment]
Not really, but I'll do a spike today and hopefully it'll be easy to add long breaks.

**Comment by dkandalov** [comment]
Btw, there is long break now. 
If you're still interested, you can try and see if the implementation does what you want.

(Sorry, it took ages to do the feature.)
